src/components/calendar/calendar.ts

```diff
@@ -141,6 +141,9 @@ export class SbbCalendarElement<T = Date> extends LitElement {
   /** A function used to filter out dates. */
   @property({ attribute: 'date-filter' }) public dateFilter?: (date: T | null) => boolean;
 
+  /** A specific date for the current datetime (timestamp in milliseconds). */
+  @property({ attribute: 'now' }) public dataNow?: number;
```

Comment:
```suggestion
  @property() public now?: number;
```

knowing this could have some conflicts in certain components, but I think dataNow doesn't make sense. It should maybe be called dateNow (but then also the attribute). but simply "now" would be my favorite :)

Reply:
I went for 'now' too